rest/runProxyMethod.ts

```diff
@@ -13,7 +13,7 @@ export async function runProxyMethod<T = any>(
   body?: unknown,
   retryCount = 0,
   bucketId?: string,
-): Promise<ProxyMethodResponse<SnakeCasedPropertiesDeep<T>>> {
+): Promise<Omit<(RestRequestResponse | RestRequestRejection), "body"> & { body?: Record<string, unknown> }> {
```

Comment:
Maybe these type declarations should be changed.

Reply:
Keep the type. If you need, then just remove the generic from it.